lib/ord.js

```diff
@@ -1,25 +1,22 @@
-const path = require("path");
-const cds = require("@sap/cds");
-const { exists } = cds.utils;
-const _ = require("lodash");
-const defaults = require("./defaults");
+const { ORD_RESOURCE_TYPE } = require('./constants');
 const {
     createAPIResourceTemplate,
+    createEntityTypeTemplate,
     createEventResourceTemplate,
-    createGroupsTemplateForService,
-    createEntityTypeTemplate
+    createGroupsTemplateForService
 } = require('./templates');
+const cds = require("@sap/cds");
+const defaults = require("./defaults");
 const { extendCustomORDContentIfExists } = require('./extendOrdWithCustom');
-const { ORD_RESOURCE_TYPE } = require('./constants');
-const { getRFC3339Date } = require('./date')
-
-const logger = cds.log('ord-plugin');
+const { getRFC3339Date } = require('./date');
+const _ = require("lodash");
+const { join } = require("path");
 
 const initializeAppConfig = (csn) => {
-    let packagejsonPath = path.join(cds.root, 'package.json')
+    let packageJsonPath = join(cds.root, 'package.json')
     let packageJson;
-    if (exists(packagejsonPath)) {
-        packageJson = require(packagejsonPath);
+    if (cds.utils.exists(packageJsonPath)) {
```

Comment:
This is good example! `cds.utils.exists` gives me the clear context. I wish `path.join` is as well.

Reply:
I will revert the change then. 👍